.circleci/config.yml

```diff
@@ -967,9 +967,16 @@ jobs:
     <<: *base_ubuntu2204_small
     steps:
       - checkout
+      - run:
+          # TODO: Add these to the base image
```

Comment:
I created an issue to keep track of this: https://github.com/ethereum/solidity/issues/14847